### DISCUSSION TITLE
New command suggestion: vote randomize team

### DISCUSSION THREAD

**Discussion by wolf109909** [ScureX/Takyon.PlayerVote#14] [Ideas]
This idea comes from a player in our community. He think the auto-balance mod is too harsh on changing side while the game is running. maybe you can make a vote version of auto-balance in the mod so players themselves can control the balance

**Comment by ScureX** [comment]
**v3.0.0** has already a **!balance** vote, which balances teams by **K/D**, as well as **!switch** which lets players switch teams a limited amount per match. Is this what you're going for? Future versions might include !shuffle for true random teams but there's already a mod that auto-shuffles.
